### ISSUE TITLE
How to convert vox to kvx?

### ISSUE THREAD

**Issue by IgeNiaI** [sirjuddington/SLADE#1457]
**SLADE Version**: (3.2.1)
**Operating System**: (Windows)

**Issue Details**:
The 3.2.0 changelog mentions .vox to .kvx conversion, but I can't find this option whatsoever. Also, when I open a package with .vox file, Slade doesn't recognize it's type.

**Comment by sirjuddington** [comment]
What kind of .vox file are you trying to open? I think the .vox support in SLADE is for a game format (I'm not 100% sure what game it is tbh, wasn't added by me), if you are using MagicaVoxel .vox models they aren't supported by SLADE

**Comment by IgeNiaI** [comment]
Yes, I was using MagicaVoxel .vox files, that explains it. Both Zandronum and GZDoom support that file though, so would be nice to have it in Slade too.
